test/rcheevos/test_condition.c

```diff
@@ -372,6 +372,7 @@ void test_condition(void) {
   TEST_PARAMS5(test_parse_condition, "A:0xH1234/8", RC_CONDITION_ADD_SOURCE, RC_OPERAND_ADDRESS, RC_OPERATOR_DIV, 0);
   TEST_PARAMS5(test_parse_condition, "A:0xH1234&8", RC_CONDITION_ADD_SOURCE, RC_OPERAND_ADDRESS, RC_OPERATOR_AND, 0);
   TEST_PARAMS5(test_parse_condition, "A:0xH1234^8", RC_CONDITION_ADD_SOURCE, RC_OPERAND_ADDRESS, RC_OPERATOR_XOR, 0);
+  TEST_PARAMS5(test_parse_condition, "A:0xH1234%8", RC_CONDITION_ADD_SOURCE, RC_OPERAND_ADDRESS, RC_OPERATOR_MOD, 0);
```

Comment:
Please also add some tests for processing the logic. Here's some examples for other operators in test_value:
https://github.com/RetroAchievements/rcheevos/blob/3decda081f98582e02d583474048dade799edb2a/test/rcheevos/test_value.c#L534-L539

Reply:
Added a handful.  Missing something needed to run the tests. Will need to sort that out.

Reply:
Just didn't realize I run the tests buy running in debugger.  Corrected a few. Added some with floats. Should be ready. Let me know if there's anything else I ought to test.

Reply:
Added typed value tests for modulus.